src/App.js

```diff
@@ -49,13 +49,13 @@ function App() {
               <Recipe recipesArray={recipesArray} />
             </Route>
             <Route exact path="/breakfast/:slug">
```

Comment:
You don't need to define all your categories manually.
You can just write `<Route path='/:category/:slug'>`